src/app/modules/time-clock/components/project-list-hover/project-list-hover.component.ts

```diff
@@ -88,6 +99,7 @@ export class ProjectListHoverComponent implements OnInit, OnDestroy {
       start_date: new Date().toISOString(),
       timezone_offset: new Date().getTimezoneOffset(),
       technologies: [],
+      activity_id: this.activities[0].id,
```

Comment:
In this index you can use 
`import { head } from 'lodash';`
`activity_id: _.head(this.activities.id)`  OR `activity_id:  _.head(this.activities).id`

Try with this options.